Code/DistGeom/DistGeomUtils.cpp

```diff
@@ -347,13 +348,22 @@ ForceFields::ForceField *construct3DForceField(
     }
   }
 
-  // minimum distance for all other atom pairs
-  fdist = etkdgDetails.boundsMatForceScaling * 10.0;
+  // minimum distance for all other atom pairs that aren't constrained
   for (unsigned int i = 1; i < N; ++i) {
     for (unsigned int j = 0; j < i; ++j) {
       if (!atomPairs[j * N + i]) {
+        fdist = etkdgDetails.boundsMatForceScaling * 10.0;
         double l = mmat.getLowerBound(i, j);
         double u = mmat.getUpperBound(i, j);
+        if (etkdgDetails.constrainedAtoms.size() &&
```

Comment:
```suggestion
        if (!etkdgDetails.constrainedAtoms.empty() &&
```
I find the above clearer. Feel free to ignore the suggestion.